src/test/java/com/google/api/codegen/testdata/java_test_library.baseline

```diff
@@ -650,6 +657,105 @@ public class LibraryServiceTest {
     Assert.assertEquals(indexMap, actualRequest.getIndexMap());
   }
 
+  public void streamShelvesTest() {
+    StreamShelvesResponse expectedResponse = StreamShelvesResponse.newBuilder().build();
+    List<GeneratedMessageV3> expectedResponses = new ArrayList<>();
+    expectedResponses.add(expectedResponse);
+    mockLibraryService.setResponses(expectedResponses);
+
+
+    MockStreamObserver<StreamingRecognizeResponse> responseObserver = new MockStreamObserver<>();
+
+    StreamingApiCallable<StreamShelvesRequest, StreamShelvesResponse> callable =
+        api.streamShelvesCallable();
+    callable.serverStreamingCall(request, responseObserver);
+
+    StreamShelvesRequest request =
+        StreamShelvesRequest.newBuilder().build();
+    requestObserver.onNext(request);
+    requestObserver.onCompleted();
```

Comment:
Sorry I was running the tests against the speech API which does not cover this server streaming case. I have tested the code against the BigTable API and fixed these issues. Consider it done.